src/content/docs/en/recipes/call-endpoints.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Call endpoints from the server
+description: Learn how to call endpoints from the server in Astro.
+i18nReady: true
+type: recipe
+---
+
+## Recipe
+
+1. Create an endpoint (e.g. `src/pages/api/hello.ts`):
+
+    ```ts title="src/pages/api/hello.ts"
+    import type { APIRoute } from 'astro'
+
+    export const get: APIRoute = () => {
+      return {
+        body: JSON.stringify({
+          hello: 'world',
+        }),
+      }
+    }
+    ```
+
+2. Import the method from the endpoint & call it from a page (e.g. `src/pages/index.astro`). Await the result in the case it's a promise, and convert it to a `Response` if it's not already. Then, you can use the `Response` methods to get the data you need:
```

Comment:
```suggestion
2. On any Astro page  (e.g. `src/pages/index.astro`), import the `get()` method from the endpoint. Then, you can use the `Response` methods to get the data you need:

  ```astro title="src/pages/index.astro"
  ---
  import { get } from './api/hello.ts'
  let response = await get(Astro)
  const data = await response.json()
  ---
  <h1>{data.greeting} world!</h1>
  ```

```
OK, there was a lot going on in that instruction. AND, since this is YOUR recipe, you should know whether or not it's a promise/response? So, I'm going to suggest cutting that, making the example more simple. 

You could include another step that showing the full example saying that depending on how much is known about the data you're fetching, you could also add a line to convert to a response, if it's not already one.

Also, we haven't explained what "Astro" is in that example above?